snap/local/runtime-helpers/bin/mongod-wrapper.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash -e
+
+# check the mongo database path
+MONGO_DATA_DIR="$SNAP_DATA"/mongo/db
+if [ -e "$MONGO_DATA_DIR" ] ; then
+    rm -rf "${MONGO_DATA_DIR:?}"/*
```

Comment:
We also fixed this in delhi. Definitely don't want to blow away the db on every restart!

Reply:
Do you want me to incorporate your fix for this into this PR? I was trying to keep the forward-porting PR's more or less in line with how we did them for Delhi, but since I already pull in a couple of commits from other PR's to fix bugs it's probably okay to also pull that one in too

Reply:
I'm fine either way, just didn't want it to get lost in the shuffle...

Reply:
I've include it here for posterity.